doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
Conceptually that seems okay to me.  However, the conduit concept is specific to GASNet. comm=ugni doesn't have it and our past (and potential future) comm=mpi also doesn't. The coming comm=ofi might or might not have something like it for its so-called fabric provider. So maybe instead of Infiniband and UDP conduit docs we want just one GASNet doc that talks about all the GASNet conduits, and perhaps in the future an ofi one that talks about all its fabric providers. But if the decision is to document the conduits individually I won't fight that consensus.